bin/plugin/commands/packages.js

```diff
@@ -152,30 +154,42 @@ async function updatePackageChangelogs(
 				}
 			}
 			const packageName = `@wordpress/${
-				changelogFile.split( '/' ).reverse()[ 1 ]
+				changelogPath.split( '/' ).reverse()[ 1 ]
 			}`;
-			const { version } = readJSONFile(
-				changelogFile.replace( 'CHANGELOG.md', 'package.json' )
+			const packageJSONPath = changelogPath.replace(
+				'CHANGELOG.md',
+				'package.json'
+			);
+			const packageLockPath = changelogPath.replace(
+				'CHANGELOG.md',
+				'package-lock.json'
 			);
-			const nextVersion =
+			const { version } = readJSONFile( packageJSONPath );
+			let nextVersion =
 				versionBump !== null
 					? semver.inc( version, versionBump )
 					: null;
 
+			if ( nextVersion !== null && isRC ) {
```

Comment:
We can’t use the actual release number because Lerna uses npm version command internally that always has to bump version but we could use `-pre.1` suffix that we could simply update to the final version when the actual publishing happens. I didn’t think about it earlier. Great idea, thanks ❤️

So, we would need two versions, one for the changelog and one for `package.json`.